docs/snippets/vue/my-component-story-configure-viewports.js.mdx

```diff
@@ -0,0 +1,29 @@
+```js
+// Button.stories.js
+
+import Button from './Button';
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
+
+export default {
+  title: 'Button',
+  parameters: {
+    // the viewports object from the Essentials addon
+    viewport: {
+      // the viewports you want to use
+      viewports: INITIAL_VIEWPORTS,
+      // your own default viewport
+      defaultViewport: 'iphone6',
+    },
+  },
+};
+
+export const Primary = () => ({
+  components: { Button },
+  template: '<Button primary label="Button" />',
+});
+Primary.parameters = {
+  viewport: {
+    defaultViewport: 'iphonex',
+  },
+};
```

Comment:
@andre-brdoch can you update the story based on what's already present for the react version of the [docs](https://storybook.js.org/docs/react/essentials/viewport) ? Just update the story name and the template to match if you don't mind

Reply:
done! I noticed that the react example uses [`MINIMAL_VIEWPORTS`](https://github.com/storybookjs/storybook/blob/master/addons/viewport/src/defaults.ts#L135), however, that one does not include iphone6 and iphonex, so I changed it to `INITIAL_VIEWPORTS`, I hope that's fine. Wasn't sure if I should edit the react example also? 😄

Reply:
it's fine. I just checked and it's ok. The minimal viewports are well...minimal :P! And don't go into to much detail. Sorry for misleading you and glad that you picked on it. 

I'm ok with you making a follow up pull request for the react snippets so that we don't get sidetracked. The snippets are in the `snippets/react` folder the file names are the same :)! 

When and if you're able to create the follow up pull request add me as a reviewer if you don't mind.